protobuf.lua

```diff
@@ -0,0 +1,612 @@
+    -- configuration { "ios_armv7_release" }
+    -- -------------------------------------------------------------
+
+    -- common configuration settings
+
+    dofile (_BUILD_DIR .. "/static_ios_armv7_release.lua")
+
+    -- project specific configuration settings
+
+    -- configuration { "ios_armv7_release" }
+
+    -- -------------------------------------------------------------
+    -- configuration { "ios_sim_debug" }
+    -- -------------------------------------------------------------
+
+    -- common configuration settings
+
+    dofile (_BUILD_DIR .. "/static_ios_sim_debug.lua")
+
+    -- project specific configuration settings
+
+    -- configuration { "ios_sim_debug" }
+
+    -- -------------------------------------------------------------
+    -- configuration { "ios_sim_release" }
+    -- -------------------------------------------------------------
+
+    -- common configuration settings
+
+    dofile (_BUILD_DIR .. "/static_ios_sim_release.lua")
+
+    -- project specific configuration settings
+
+    -- configuration { "ios_sim_release" }
+
+    -- -------------------------------------------------------------
+    -- configuration { "ios_arm64_debug" }
+    -- -------------------------------------------------------------
+
+    -- common configuration settings
+
+    dofile (_BUILD_DIR .. "/static_ios_arm64_debug.lua")
+
+    -- project specific configuration settings
+
+    -- configuration { "ios_arm64_debug" }
+
+    -- -------------------------------------------------------------
+    -- configuration { "ios_arm64_release" }
+    -- -------------------------------------------------------------
+
+    -- common configuration settings
+
+    dofile (_BUILD_DIR .. "/static_ios_arm64_release.lua")
+
+    -- project specific configuration settings
+
+    -- configuration { "ios_arm64_release" }
+
+    -- -------------------------------------------------------------
+    -- configuration { "ios_sim64_debug" }
+    -- -------------------------------------------------------------
+
+    -- common configuration settings
+
+    dofile (_BUILD_DIR .. "/static_ios_sim64_debug.lua")
+
+    -- project specific configuration settings
+
+    -- configuration { "ios_sim64_debug" }
+
+    -- -------------------------------------------------------------
+    -- configuration { "ios_sim64_release" }
+    -- -------------------------------------------------------------
+
+    -- common configuration settings
+
+    dofile (_BUILD_DIR .. "/static_ios_sim64_release.lua")
+
+    -- project specific configuration settings
+
+    -- configuration { "ios_sim64_release" }
+
+    -- -------------------------------------------------------------
+  end
+
+  if (_OS_IS_ANDROID) then
+    -- -------------------------------------------------------------
+    -- configuration { "android" } == _OS_IS_ANDROID
+    -- -------------------------------------------------------------
+
+    -- common configuration settings
+
+    defines {
```

Comment:
This needs to be indented and go below the configuration { android } block below.  The configuration needs to be uncommented.